daemon/cmd/daemon_main.go

```diff
@@ -1094,7 +1094,7 @@ func initializeFlags() {
 	flags.Bool(option.ExternalClusterIPName, false, "Enable external access to ClusterIP services (default false)")
 	option.BindEnv(option.ExternalClusterIPName)
 
-	flags.IntSlice(option.VLANBPFBypass, []int{}, "List of explicitly allowed VLAN IDs, '0' id will allow all VLAN IDs")
```

Comment:
Would it make sense to add a comment briefly describing this workaround and a link to the your PR?

Reply:
Sure, it may be useful. I'll fix it, thank you!